e2e-tests/src/index.ts

```diff
@@ -95,13 +95,20 @@ app.ws("/:pathName", (ws, req) => {
 });
 
 app.get("/:documentPath/", (req, res) => {
-  const html = `<html><script src="http://${
+  const html = `<html><script src="${req.secure ? "https" : "http"}://${
     req.hostname
-  }:28891/index.js?defineGlobals=true&websocketUrl=${getWebsocketUrl(req)}"></script></html>`;
+  }${req.secure ? "/client/" : ":28891/"}/index.js?defineGlobals=true&websocketUrl=${getWebsocketUrl(req)}"></script></html>`;
```

Comment:
If this server is serving the `/client/` then I don't think it should revert to using the server on `28891` if the request isn't secure. I think the problem here might be that the `req.hostname` doesn't include this server's port number, which is solvable by parsing the url rather than just the `req.hostname`.